src/benchmark/benchmark_join_fixture.cpp

```diff
@@ -36,10 +38,15 @@ void BenchmarkJoinFixture::SetUp(::benchmark::State& state) {
   }
 
   // TODO(anyone): replace with EncodingType::Dictionary once joins (especially index join) support these
+  const auto left_chunk_size = static_cast<ChunkID>(state.range(0) / 50);
```

Comment:
Is there a reason why you increased the number of chunks for all Join benchmarks from 4 to 50?

Reply:
I wanted to have multiple chunks per NUMA Node on pella. The number of 50 was chosen based on the presentations we saw at the beginning of the semester where I remember 50 being a "reasonable" number of chunks to expect.

Reply:
Makes sense. I am just pointing it out because this is a change that applies for all Join benchmarks, not just the one you added. Which is probably fine, just worth noting.